File: app/Models/City.php
<?php

namespace App\Models;

use Laravel\Sanctum\HasApiTokens;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class City extends Model
{
    use HasApiTokens, HasFactory, Notifiable;
    protected $fillable = [
        'name',
        'minimum_free_delivery_order_amount',
        'delivery_charges'
    ];
}

File: customer_app/lib/ui/profile/address/models/city.dart
class City {
  int? id;
  String? name;
  int? minimumFreeDeliveryOrderAmount;
  int? deliveryCharges;

  City(
      {this.id,
      this.name,
      this.minimumFreeDeliveryOrderAmount,
      this.deliveryCharges});

  City.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    minimumFreeDeliveryOrderAmount = json['minimum_free_delivery_order_amount'];
    deliveryCharges = json['delivery_charges'];
  }


}
